Marlin/src/module/polargraph.cpp

```diff
@@ -44,7 +44,7 @@ xy_pos_t draw_area_min = { X_MIN_POS, Y_MIN_POS },
 
 xy_float_t draw_area_size = { X_MAX_POS - X_MIN_POS, Y_MAX_POS - Y_MIN_POS };
 
-float polargraph_max_belt_len = HYPOT(draw_area_size.x, draw_area_size.y);
+float polargraph_max_belt_len = POLARGRAPH_MAX_BELT_LEN;
```

Comment:
None of these configurable values is being saved to EEPROM, so code to load/save/reset should be added to `settings.cpp`.